src/main/java/core/basesyntax/service/impl/DataConverterServiceImpl.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataConverterService;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterServiceImpl implements DataConverterService {
+    public static final String SEPARATOR = ",";
+    public static final int ELEMENTS_IN_A_ROW = 3;
+
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> data) {
+        List<FruitTransaction> transactions = new ArrayList<>();
+
+        for (int i = 1; i < data.size(); i++) {
```

Comment:
The loop starts from index 1, which assumes that the first line of the data is a header. Ensure that this assumption is documented or handled elsewhere if the input data format changes.